### ISSUE TITLE
Missing page titles

### ISSUE THREAD

**Issue by shiffman** [shiffman/shiffman.net#62]
Page titles should be added:

<img width="326" alt="screen shot 2018-10-19 at 2 24 42 pm" src="https://user-images.githubusercontent.com/191758/47236650-cce10480-d3aa-11e8-86ee-30c08bb6a2e6.png">

(Error from https://shiffman.net/a2z/)

**Comment by phucduong86** [comment]
Hi Daniel,

I'm hoping to work on this issue but not too sure how the site is built (I saw a wordpress folder but it looks more like a static generated site?).
The tittle would need to be included in the _includes/head.html template and be dynamic base on which page the user is on.

Thank you

**Comment by shiffman** [comment]
The website is built with jekyll / github pages. I should really create a CONTRIBUTING guide. The wordpress folder is old stuff just there for archival purposes.

**Comment by phucduong86** [comment]
I believe this issue can be closed.
